src/CollectdWinService/WindowsPerformanceCounterPlugin.cs

```diff
@@ -97,7 +106,7 @@ public MetricValue GetMetricValue(List<double> vals)
         {
             var metricValue = new MetricValue
             {
-                HostName = s_hostName,
+                HostName = String.IsNullOrEmpty(InstanceName) ? s_hostName : InstanceName,

```

Comment:
instead of getting it from this file, can you move this login into [here](https://github.com/bloomberg/collectdwin/blob/master/src/CollectdWinService/Util.cs#L20-L32)?